service/hook/assembla/assembla.go

```diff
@@ -0,0 +1,140 @@
+package assembla
+
+//
+// Docs: https://articles.assembla.com/assembla-basics/learn-more/post-information-to-external-systems-using-webhooks
+//
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/bitrise-io/bitrise-webhooks/bitriseapi"
+	hookCommon "github.com/bitrise-io/bitrise-webhooks/service/hook/common"
+)
+
+// --------------------------
+// --- Webhook Data Model ---
+
+// SpaceEventModel ...
+type SpaceEventModel struct {
+	Space	string	`json:"space"`
+	Action	string	`json:"action"`
+	Object	string	`json:"object"`
+}
+
+// MessageEventModel ...
+type MessageEventModel struct {
+	Title   string	`json:"title"`
+	Body    string	`json:"body"`
+	Author	string	`json:"author"`
+}
+
+// GitEventModel ...
+type GitEventModel struct {
+	RepositorySuffix	string      `json:"repository_suffix"`
+	RepositoryURL		string      `json:"repository_url"`
+	Branch				string      `json:"branch"`
+	CommitID			string      `json:"commit_id"`
+}
+
+// PushEventModel ...
+type PushEventModel struct {
+	SpaceEventModel	    SpaceEventModel		`json:"assembla"`
+	MessageEventModel	MessageEventModel	`json:"message"`
+	GitEventModel		GitEventModel		`json:"git"`
+}
+
+// ---------------------------------------
+// --- Webhook Provider Implementation ---
+
+// HookProvider ...
+type HookProvider struct{}
+
+func detectContentType(header http.Header) (string, error) {
+	contentType := header.Get("Content-Type")
+	if contentType == "" {
+		return "", errors.New("No Content-Type Header found")
+	}
+
+	return contentType, nil
+}
+
+func detectAssemblaData(pushEvent PushEventModel) error {
+	if (pushEvent.GitEventModel.CommitID == "---") ||
```

Comment:
I'd also suggest you to check whether these are empty (e.g. `pushEvent.GitEventModel.CommitID == ""`) in addition to the `"---"` value